test/Datadog.Trace.ClrProfiler.IntegrationTests/SmokeTests/SmokeTestBase.cs

```diff
@@ -79,7 +79,7 @@ protected void CheckForSmoke(bool shouldDeserializeTraces = true)
                 {
                     // Initialize StartInfo
                     process.StartInfo.FileName = executable;
-                    EnvironmentHelper.SetEnvironmentVariables(agentPort, aspNetCorePort, executable, process.StartInfo.EnvironmentVariables);
+                    EnvironmentHelper.SetEnvironmentVariables(agentPort, aspNetCorePort, executable, false, process.StartInfo.EnvironmentVariables);
```

Comment:
nit, for readability
```suggestion
                    EnvironmentHelper.SetEnvironmentVariables(agentPort, aspNetCorePort, executable, netStandardEnabled: false, process.StartInfo.EnvironmentVariables);
```

Reply:
Actually, I don't know if this will compile without adding the name for the next parameter as well. I think it depends on C# version?